__tests__/app.test.js

```diff
@@ -82,6 +82,25 @@ describe("/api/articles/:article_id", () => {
       });
   });
 
+  test("GET 200 Responds with a total count of of all the comments with the passed article id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article_id).toBe(1);
```

Comment:
We've previously tested for all these properties so no need to duplicate, we can just write a new test for the new comment_count property 👍